stores/healthcareProfessionalsStore.ts

```diff
@@ -12,6 +17,17 @@ export const useHealthcareProfessionalsStore = defineStore(
         const healthcareProfessionalsData: Ref<HealthcareProfessional[]>
         = ref([])
         const selectedHealthcareProfessionalId: Ref<string> = ref('')
+        const healthcareProfessionalSectionFields = {
```

Comment:
I think we could improve this design a bit to follow better vue practices. Typically you want to reference the `healthcareProfessionalsData`  in your component and have local `ref`s version that binds to what is in the store. 
You essentially did this with this `healthcareProfessionalSectionFields` object, so I think you can just move this within the component then.

Reply:
@ermish These sections in the future can be used within the `edit-healthcare-professional` or `edit-submission` Would it not be better to keep in the store then?